java/ray.config.ini

```diff
@@ -62,9 +62,9 @@ deploy = false
 
 onebox_delay_seconds_before_run_app_logic = 0
 
-use_raylet = false
+use_raylet = true
 
-static_resources = CPU:4,GPU:0
+static_resources = CPU:4,GPU:0,ResourceA:10,ResourceB:8
```

Comment:
If there're spaces between items, e.g., `CPU:4, GPU:0, ResourceA:10,   ResourceB:8`, can we still parse it correctly?

Reply:
It will parse the string correctly now.